hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/security/acl/IAccessAuthorizer.java

```diff
@@ -114,6 +121,14 @@ public static ACLType getACLRight(String type) {
 
     }
 
+    public static BitSet getACLBitSet(byte[] acls) {
+      return BitSet.valueOf(acls);
+    }
+
+    public static String getACLString(byte[] acls) {
+      return getACLString(BitSet.valueOf(acls));
+    }
```

Comment:
I suggest moving these to `OzoneAcl` and making them private.  This would make it easier to completely get rid of `BitSet` in the future by reducing the public interface.